features/step_definitions/generate.rb

```diff
@@ -40,9 +40,20 @@
 Then("the product {string} with destination {string} does not contain resource {string}") do |product, destination, resource|
   resource_path = Xcode.find_resource(
     product: product,
-    destination: destination,
-    resource: resource,
-    derived_data_path: @derived_data_path
+      destination: destination,
+      resource: resource,
+      derived_data_path: @derived_data_path
   )
   refute(resource_path)
 end
+
+Then("the product {string} with destination {string} has an entry of {string} in the info.plist with the value of {string}") do |product, destination, setting, value|
+  plist_path = Xcode.info_plist_for_product_with_name(
+    product: product,
+      destination: destination,
+      derived_data_path: @derived_data_path
+  )
+  plist = CFPropertyList::List.new(file: plist_path)
+  data = CFPropertyList.native_types(plist.value)
```

Comment:
I think you need to add the following at the top:

```
require 'cfpropertylist'
```

Unless Cucumber requires the external gems automatically, which I doubt.